### ISSUE TITLE
Integration does not appear after HACS install on HA core

### ISSUE THREAD

**Issue by Toets** [mammuth/ha-fritzbox-tools#106]
I followed the prereqs for install on Home Assistant core. After HACS install the fritzbox_tools does not appear in the Integrations pane. As alternative I edited the config manually. Now I see the following in my HA log: [homeassistant.bootstrap] Waiting on integrations to complete setup: fritzbox_tools

In my router log I see HA logging in but I am unsure if this is caused by fritzbox_tools or fritzbox integration  I previously configured  but put to "ignore":

router log: App belonging to user USER logging in from IP address W.X.Y.Z.

arch | x86_64
-- | --
dev | false
docker | false
hassio | false
installation_type | Home Assistant Core
os_name | Linux
os_version | 4.15.0-112-generic
python_version | 3.7.5
timezone | Europe/Amsterdam
version | 0.113.3
virtualenv | true

**Comment by mammuth** [comment]
You could enable debug logging for fritzbox_tools in the HA config and restart HA. Maybe this brings information what's going on.

**Comment by chemelli74** [comment]
This custom integration is now deprecated because is fully merged into HomeAssistant core.

I would suggest you configure the new integration and retest your issue; if still present then open a issue on HomeAssistant Core repository.

Simone
